Tests/Clappr_Tests/Classes/Plugin/Playback/AVFoundationPlaybackTests.swift

```diff
@@ -780,6 +780,127 @@ class AVFoundationPlaybackTests: QuickSpec {
                             expect(startAtValue).toEventually(beCloseTo(10, within: 0.1))
                         }
                     }
+                    
+                    context("with liveStartTime") {
+                        
+                        func setupLiveDvr(liveStartTime: Double)  {
+                            playback = AVFoundationPlayback(options: [kMinDvrSize: 0, kLiveStartTime: liveStartTime])
+                            
+                            asset = AVURLAssetStub(url: URL(string:"globo.com")!)
+                            asset.set(duration: .indefinite)
+                            
+                            item = AVPlayerItemStub(asset: asset)
+                            item.setWindow(start: 0, end: 1000)
+                            item.set(currentTime: CMTime(seconds: 1000, preferredTimescale: 1))
+                            item.set(currentDate: Date(timeIntervalSince1970: 3000))
+                            
+                            player = AVPlayerStub()
+                            player.set(currentItem: item)
+                            
+                            playback.player = player
+                            player.setStatus(to: .readyToPlay)
+                        }
+                        
+                        context("when video is live with dvr") {
+                            context("and liveStartTime is between seekable range") {
+                                it("triggers didUpdatePosition") {
+                                    setupLiveDvr(liveStartTime: 2500.0)
+                                    
+                                    var didCallUpdatePosition = false
+                                    var liveStartTime: Double = 0
+                                    
+                                    playback.on(Event.didUpdatePosition.rawValue) { userInfo in
+                                        didCallUpdatePosition = true
+                                        liveStartTime = userInfo!["position"] as! Double
+                                    }
+                                    
+                                    playback.handleDvrAvailabilityChange()
+                                    
+                                    expect(didCallUpdatePosition).toEventually(beTrue())
+                                    expect(liveStartTime).toEventually(equal(500))
```

Comment:
Hey guys, I found a bit hard to understand why the result would be 500. Do you think is there a more descriptive way to write this test?

My suggestion would be to pass a `currentDate` to `setupLiveDvr`, so the math to calculate the result (expected value) would be contained inside the test itself. WDYT?